src/clr-angular/data/datagrid/datagrid.spec.ts

```diff
@@ -778,6 +815,16 @@ export default function(): void {
       });
 
       describe('View', function() {
+        /*
+         * For some reason this test is breaking all other tests. 
+         * Not sure why - need to investigate more 
+         * 
+        it('should have aria-label with value `Select`', function() {
```

Comment:
I'm not overly worried about this test, but it would better to do `xit` instead of commenting it out.

Reply:
I know, but even when it `xit` i get this error

```
HeadlessChrome 75.0.3770 (Mac OS X 10.14.5) ERROR
  {
    "message": "An error was thrown in afterAll\nUncaught TypeError: Cannot read property 'nativeElement' of undefined thrown",
    "str": "An error was thrown in afterAll\nUncaught TypeError: Cannot read property 'nativeElement' of undefined thrown"
  }

```

Something is very wrong with these lines. I try even writing them again just to be sure that there is no strange symbol.